src/oge/data_cleaning.py

```diff
@@ -49,12 +49,27 @@
 def clean_eia923(
     year: int,
     small: bool,
-    add_subplant_id: bool = True,
     calculate_nox_emissions: bool = True,
     calculate_so2_emissions: bool = True,
-):
-    """This is the coordinating function for cleaning and allocating generation and fuel
-    data in EIA-923."""
+) -> tuple[pd.DataFrame, pd.DataFrame, pd.DataFrame]:
+    """This is the coordinating function for cleaning and allocating generation and
+    fuel data in EIA-923.
+
+    Args:
+        year (int): year to consider.
+        small (bool): should a subset of data be considered.
+        calculate_nox_emissions (bool, optional): whether or not clculate NOx emission
+            from fuel consumption. Defaults to True.
+        calculate_so2_emissions (bool, optional): whether or not clculate So2 emission
+            from fuel consumption. Defaults to True.
+
+    Returns:
+        tuple[pd.DataFrame, pd.DataFrame, pd.DataFrame]: the first data frame is the
+            generation and fuel alocated with NOx and SO2 emission if requested. The
+            second data frame is the primary fuel table. Finally, the third data frame
+            encloses the emission factors. All three data frames are at the subplant
```

Comment:
The third data frame has subplant-specific emission factors, which is used for filling missing emissions data in cems later in the pipeline

Reply:
Done